app/helpers/application_helper.rb

```diff
@@ -29,6 +29,14 @@ def navigation_items
     []
   end
 
+  def show_confirmation_reminder?
+    return false unless current_user
+
+    !current_user.confirmed_at?
```

Comment:
`show_confirmation_reminder?` will only show for the initial confirmation on account registration.  The commit message mentions changing email addresses, but this won't cover that situation since `confirmed_at` will still be set with the time of clicking the first confirmation email.

If the intention was to display the message at that point, then you'll need to also check for the presence of `unconfirmed_email`, e.g. `!current_user.confirmed_at? || current_user.unconfirmed_email?`.

It may be prudent to add a test to cover the two scenarios (with both expecting to show the banner until the new email address is confirmed):
1. user with an already confirmed email changes their email address
2. user with an unconfirmed email changes their email address

Reply:
Thanks for this Bruce;
- [I have bulked out a feature test suite here](https://github.com/alphagov/govuk-account-manager-prototype/pull/436/commits/971fdaa0c2563c06e1781b66fe52691bdd623263)
- [and tweaked the logic here](https://github.com/alphagov/govuk-account-manager-prototype/pull/436/commits/177ba8891c9b708965ac83c74581a7f9783e3fd0) <- Thanks for the example on that one, very helpful!

Shout if you are happy with those then I squash this all back into one commit